src/OSPSuite.Core/Domain/SensitivityAnalyses/SensitivityAnalysisRunResult.cs

```diff
@@ -9,22 +9,39 @@ public class SensitivityAnalysisRunResult
    {
       private readonly List<PKParameterSensitivity> _allPKParameterSensitivities = new List<PKParameterSensitivity>();
 
+      /// <summary>
+      ///    This property won't be serialized
+      /// </summary>
+      private readonly List<OutputParameterSensitivity> _allOutputParameterSensitivities = new List<OutputParameterSensitivity>();
+
       public virtual IReadOnlyList<PKParameterSensitivity> AllPKParameterSensitivities => _allPKParameterSensitivities;
 
-      public virtual void AddPKParameterSensitivity(PKParameterSensitivity parameterSensitivity)
-      {
-         _allPKParameterSensitivities.Add(parameterSensitivity);
-      }
+      public virtual IReadOnlyList<OutputParameterSensitivity> AllOutputParameterSensitivities => _allOutputParameterSensitivities;
 
-      public IEnumerable<PKParameterSensitivity> AllFor(string pkParameterName, string outputPath)
+      public virtual void AddPKParameterSensitivity(PKParameterSensitivity pkParameterSensitivity) => _allPKParameterSensitivities.Add(pkParameterSensitivity);
+
+      public virtual void AddOutputParameterSensitivity(OutputParameterSensitivity outputParameterSensitivity) => _allOutputParameterSensitivities.Add(outputParameterSensitivity);
+
+      public IEnumerable<PKParameterSensitivity> AllPKParameterSensitivitiesFor(string pkParameterName, string outputPath)
       {
          return _allPKParameterSensitivities.Where(
             x => string.Equals(x.QuantityPath, outputPath) && string.Equals(x.PKParameterName, pkParameterName));
       }
 
+      public IEnumerable<OutputParameterSensitivity> AllOutputParameterSensitivitiesFor(string outputPath, string parameterPath)
+      {
+         return _allOutputParameterSensitivities.Where(
+            x => string.Equals(x.QuantityPath, outputPath) && string.Equals(x.ParameterPath, parameterPath));
+      }
+
       public PKParameterSensitivity PKParameterSensitivityFor(string pkParameterName, string outputPath, string parameterName)
       {
-         return AllFor(pkParameterName, outputPath).Find(x => string.Equals(parameterName, x.ParameterName));
+         return AllPKParameterSensitivitiesFor(pkParameterName, outputPath).Find(x => string.Equals(parameterName, x.ParameterName));
+      }
+
+      public OutputParameterSensitivity[] OutputParameterSensitivitiesFor(string outputPath, string parameterPath)
```

Comment:
This will be exposed to R. For one output and one parameter (path), gives me all possible variations (one for each value of the parameter). 
Note that the value for the base simulation is not returned